src/extension/openFileAtLocation.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {RemoteExtension} from "../common/remoteExtension";
+import {ReactNativeProjectHelper} from "../common/reactNativeProjectHelper";
+import {InternalErrorCode} from "../common/error/internalErrorCode";
+import {ErrorHelper} from "../common/error/errorHelper";
+import * as path from "path";
+
+/* Usage:
+...path\openFileAtLocation.js filename:lineNumber
+...path\openFileAtLocation.js filename
+*/
+
+if (process.argv.length < 3) {
+    throw "Wrong number of parameters provided. Please refer to the usage of this script for proper use.";
+}
+const fileInfo = process.argv[2].split(":");
```

Comment:
This has a bug. When the path to the file is from windows (`C:\path\to\file:1`), the `split(":")` doesn't give the desired behavior. I was thinking on handling this by doing something like: 
```
if (process.argv[2].indexOf(":") == 1) {
    filename = "C:" + fileInfo[1];
    lineNumberIndex = 2;
} else {
    filename = fileInfo[0];
    lineNumberIndex = 1;
}

if (fileInfo.length >= 2) {
    lineNumber = parseInt(fileInfo[lineNumberIndex], 10);
}
```
What do you guys think? @digeff @MSLaguana